tests/shipping-preview/models/Delivery - Boleto.model.js

```diff
@@ -0,0 +1,64 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  fillEmail,
+  getRandomEmail,
+  fillProfile,
+} from '../../../utils/profile-actions'
+import {
+  goToPayment,
+  fillShippingInformation,
+} from '../../../utils/shipping-actions'
+import { payWithBoleto, completePurchase } from '../../../utils/payment-actions'
+import { ACCOUNT_NAMES, SKUS } from '../../../utils/constants'
+
+export default function test(account) {
+  xdescribe(`Delivery - Boleto - ${account}`, () => {
+    before(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with only delivery', () => {
+      const email = getRandomEmail()
+
+      setup({ skus: [SKUS.DELIVERY_MULTIPLE_SLA], account })
+      fillEmail(email)
+      fillProfile()
+      fillShippingInformation(account)
+      if (account === ACCOUNT_NAMES.NO_LEAN) {
+        cy.get('#shipping-data')
+          .contains('PAC')
+          .should('be.visible')
+        cy.get('#shipping-data')
+          .contains('Motoboy')
+          .should('be.visible')
+        cy.get('#shipping-data')
+          .contains('Expressa')
+          .should('be.visible')
+        cy.get('#shipping-data')
+          .contains('PAC Lento')
+          .should('be.visible')
+      } else {
+        cy.get('#shipping-data')
+          .contains('Mais rápida')
+          .should('be.visible')
+        cy.get('#shipping-data')
+          .contains('Mais econômica')
+          .should('be.visible')
+      }
+      goToPayment()
+      payWithBoleto()
+      completePurchase()
```

Comment:
I'm not sure if we should proceed to Checkout while testing Shipping Preview (unless we need to make sure its info will be available on Omnishipping), because we may end up with Shipping Preview tests failing because of "finishing purchase" errors.
What do you think? @jeffersontuc @marcoskwkm

Reply:
It makes sense. But in the other way, the intention was not only testing the `shipping-preview`, but also to reuse on `omnishipping` the information (postal code, invoice address) provided to it. And especially now that they are sharing the same `redux` and don't work as fully isolated components anymore. But I agree that a more objective e2e tests provide us more stability. I'm not yet convinced of what path we should take. What do you guys think, @vtex/checkout-ui ?

Reply:
Limiting the scope of Shipping Preview tests just to Shipping Preview itself will greatly improve the reliability of these tests, which I think is very important at the stage we are right now.

I do think we need some tests that include Shipping Preview in the whole purchase flow, especially to test the consistency between Shipping Preview and Omnishipping, but IMO that takes a lower precedence at the moment.

Reply:
It won't take much time to remodel it thought.